### DISCUSSION TITLE
Next Data modeling 06.03.24

### DISCUSSION THREAD

**Discussion by andreaa93** [FAIRmat-NFDI/AreaA-data_modeling_and_schemas#141] [Data Modeling Meeting]
Hi, 

we almost finished the issues created last Data Modeling Meeting on 21.02.24.

Some topics to discuss in the upcoming one:

- [?] ThinFilm and ThinFilmStack properties such as chamber_pressure and temperature
- [x] A TimeSeries class to be adopted to record process parameters (I have currently an example in MOVPE 1)
- [x] PrecursorPreparation aka SolutionPreparation
- [?] accessing and filling "workflow2" from eln throws errors

**Comment by andreaa93** [comment]
@hampusnasstrom this would be our discussion list from my side for Wednesday 10am

**Reply by hampusnasstrom** [reply]
Sounds great!
